cypress/support/pages/admin_console/manage/RoleMappingTab.ts

```diff
@@ -19,8 +20,8 @@ export default class RoleMappingTab {
     return this;
   }
 
-  clickAssignRole() {
-    cy.getId(this.assignRole).click();
+  clickAssignRole(notEmpty: boolean = true) {
```

Comment:
No need to specify a type here as the TypeScript compiler can infer the type from the assignment.
```suggestion
  clickAssignRole(notEmpty = true) {
```